shared/src/business/useCases/updatePetitionDetailsInteractor.old.test.js

```diff
@@ -0,0 +1,202 @@
+const {
+  CASE_STATUS_TYPES,
```

Comment:
Same as the other file updated for 7166, there's no new updatePetitionDetailsInteractor test file included in this PR - seems like that should be added with an updated test for the new if statement above.

Reply:
New test included to cover this code.

Reply:
Thanks!